server/utils/testUtils/index.js

```diff
@@ -1,47 +1,17 @@
 import isNil from 'lodash/isNil';
 
-const testDate = new Date();
-const addressesTable = {
```

Comment:
Are these not used for other tests?

Reply:
Moved these into a separate file in mockData.

Reply:
Moved all the mocks into a separate file.